projects/module-1/hello-worlds.md

```diff
@@ -0,0 +1,261 @@
+---
+title: Hello Worlds - Pair
+---
+
+## Learning Goals
+
+* Write clean, DRY JavaScript
+  * Build out functionality using functions that show trends toward SRP
+  * Manipulate the page after it has loaded adding, removing, and updating elements on the DOM
```

Comment:
Not sure what this sentence is saying. Is it meant to be two parts?

"Manipulate the page after it has loaded - adding, removing, and updating elements on the DOM"

Reply:
Yes - I'll add clarifying punctuation! I just copy pasted it from another M1 project :sweat_smile: